runtime/oti/j9nonbuilder.h

```diff
@@ -4923,6 +4923,9 @@ typedef struct J9VMThread {
 	U_32 riPadTo8;
 #endif /* !J9VM_ENV_DATA64 */
 #endif /* J9VM_JIT_RUNTIME_INSTRUMENTATION */
+#if defined(J9VM_ZOS_3164_INTEROPERABILITY)
+	U_32 jniEnv31;
```

Comment:
This will misalign the rest of the structure.

Reply:
Added U_32 pad member immediately after.  Since `J9VM_ZOS_3164_INTEROPERABILITY` applies to 64-bit, I didn't see how to optimize it with the prior paddings via ifdef's, but left a comment.  https://github.com/eclipse/openj9/pull/12074/commits/1bb3113393e6c199034b4b023b886b6a656b2ee7